website/docs/Ganglion/02-Ganglion.md

```diff
@@ -88,3 +88,8 @@ _Inner Layer 2_
 ![Ganglion Layer 2](../assets/GanglionImages/ganglion_layer2.png)  
 _Bottom Layer_  
 ![Ganglion Bottom Layer](../assets/GanglionImages/ganglion_bottom.png)  
+
+### Ganglion Board PCB Files
+
+[OpenBCI Ganglion Board Gerber Files](https://github.com/OpenBCI/Ganglion_Hardware_Design_Files/tree/master/OBCI_Ganglion_Plots)
+[OpenBCI Ganglion Board BOM](https://github.com/OpenBCI/Ganglion_Hardware_Design_Files/blob/master/Ganglion_01_BOM.xlsx)
```

Comment:
@tharun-OpenBCI You can add an extra line between these 94 and 95 and these two links should display on different lines, making it clearer that there are two links

Reply:
Oh okay. Thanks for the suggestion. It's done. I pushed the updated code to my fork